### PR TITLE
Update discourse module (Engage pages v2)

### PR DESCRIPTION
**Pull request by carkod**
## Done

Update engage pages and takeovers to new engage pages

### Engage pages
- Go to https://jp-ubuntu-com-410.demos.haus/engage
- Click on a page and check that it looks correct compared to production
- Fill in the form and check that the thank-you page renders correctly
- Check that the resource (if available) can be downloaded

### Takeovers
- Go to https://jp-ubuntu-com-410.demos.haus/takeovers. Check that active takeovers match with prod https://jp.ubuntu.com/takeovers
- Go to https://jp-ubuntu-com-410.demos.haus/ and check that the takeovers on this homepage load correctly, refresh a few times and check that it changes

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/5122

### CODE REVIEW

**Comment by carkod** [comment]
https://jp-ubuntu-com-410.demos.haus/

**Comment by carkod** [comment]
Don't merge after review, need to give a heads up to the product managers

**Review by sowasred2012 — APPROVED** [review]
LGTM - demos seem to match what is live

File: konf/site.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ domain: jp.ubuntu.com
 image: prod-comms.docker-registry.canonical.com/jp.ubuntu.com
 
 useProxy: false
+readinessPath: "/"
```
Comment (nottrobin) [inline konf/site.yaml:6]: So this is needed? Why?

**Pull request merged**